scripts/us_nces/common/dcid_existance.py

```diff
@@ -0,0 +1,131 @@
+import datacommons as dc
+import time
+import requests_cache
+import urllib
+from absl import logging
+
+dc.utils._API_ROOT = 'http://api.datacommons.org'
+
+
+def dc_api_wrapper(function,
```

Comment:
can drop this file and used the same file checked into https://github.com/datacommonsorg/data/blob/master/util/dc_api_wrapper.py